src/pages/detail-page/subpages/tab-panels/CitationPanel.tsx

```diff
@@ -70,7 +77,11 @@ const CitationPanel = () => {
       },
       {
         title: "License",
-        component: <div>License</div>,
+        component: license ? (
+          <LicenseBlock license={license} />
+        ) : (
+          <div>no license</div>
+        ),
```

Comment:
do we have any design/plan if the dataset has no particular section?

Reply:
I think it would be better to not render that part if no data

Reply:
Thank you. this consideration exists for all the part of the details. I just raised a ticket for it to implement empty block https://github.com/aodn/backlog/issues/5780            Will do it in the future